test/feature/configuration/serializer_lookup/using_serializable_resource_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+
+describe 'feature' do
+  describe 'configuration' do
+    describe 'serializer lookup' do
+      describe 'using SerializableResource' do
+        it 'uses the child serializer' do
```

Comment:
more tests need to exist in this file such that it can be the example of possible lookup chain scenarios. 

Including:
 - Versioned APIs (both via URL and request header)
 - Versioned APIs with fallback to earlier version
 - Mimicing the controller's namespace.